emailtemplateservice/tool/src/main/resources/messages.properties

```diff
@@ -0,0 +1,48 @@
+mainview.key=Template Key
```

Comment:
@mpellicer what should @victorfv2 do with these messages.properties files? He moved the files to another directory and changed some values in english, but these values weren't updated in other's languages files. Should he only upload the english translation file? Or is it ok like this?